build/dependencies.props

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
   <!-- Partner Versions-->
   <PropertyGroup>
-    <CodeAnalysisVersion>2.3.0-beta1</CodeAnalysisVersion>
+    <CodeAnalysisVersion>2.3.0-beta3-61731-06</CodeAnalysisVersion>
```

Comment:
BTW do we have confirmation from the Roslyn team that this is the exact correct build of their packages to use for the Preview 2 release?